{{cookiecutter.slug}}/package.backbone.json

```diff
@@ -22,19 +22,19 @@
     "test-back": "yarn back pytest",
     "test-func": "yarn func pytest",
     "test": "yarn test-back && yarn test-front && yarn test-func",
-    "watch-front": "yarn gulp watch --config ../config.json",
+    "watch-front": "yarn gulp watch --config config.root.json",
```

Comment:
This should be changed back, and the config file in question should be moved to the old location by the post-generation hook.